README.md

```diff
@@ -33,10 +33,10 @@ If you are migrating from another AUR helper, you can simply install Yay with th
 The initial installation of Yay can be done by cloning the PKGBUILD and
 building with makepkg:
 
-Before you begin, make sure you have the `base-devel` package group installed.
+Before you begin, make sure you have the `base-devel` package group and the `go` package installed.
 
 ```sh
-pacman -S --needed git base-devel
+pacman -S --needed git base-devel go
```

Comment:
```suggestion
pacman -S --needed --asdeps git base-devel go
```

Both `go` and `git` should be installed with `--asdeps` actually.

`base-devel` should be kept as explicit I think (at least I think it's better) but for the purpose of installing yay they are indeed make deps

Reply:
Updated as suggested

Reply:
Updated as suggested